conda-store-server/conda_store_server/server/views/api.py

```diff
@@ -280,11 +280,11 @@ def api_post_specification():
     )
 
     try:
-        api.post_specification(conda_store, specification, namespace_name)
+        build_id = api.post_specification(conda_store, specification, namespace_name)
     except ValueError as e:
         return jsonify({"status": "error", "error": str(e.args[0])}), 400
 
-    return jsonify({"status": "ok"})
+    return jsonify({"status": "ok", "data": build_id})
```

Comment:
For consistency with other endpoints
```suggestion
    return jsonify({"status": "ok", "data": {"build_id": build_id}})
```

Reply:
Err. Camel vs snake case. 😬

Note the following inconsistency:

```
curl -X PUT \
  -H 'Content-Type: application/json' \
  -d '{"buildId": 1}' \
  http://localhost:5000/api/v1/environment/gab/bag  
```

The corresponding code point: https://github.com/Quansight/conda-store/blob/main/conda-store-server/conda_store_server/server/views/api.py#L234

Reply:
It looks to me that Conda Store's GET response payloads use snake_case

Reply:
I have a preference for json payloads to use camelCase but it's not a big deal to me either way. Consistency is what I'm after : )